lib/output.js

```diff
@@ -694,6 +694,7 @@ function OutputStream(options) {
         output.semicolon();
     });
     DEFPRINT(AST_Toplevel, function(self, output){
+        use_asm = false;
```

Comment:
The issue is that if multiple trees were printed, like in tests, the var use_asm wouldn't be triggered because that is code that is executed only once, and only when generating the tree generator.